NSSNamespace/Initial_Bangazon_Site
# Es dropdowns

Database should now seed with six product subcategories (2 per general category).
Products in the database should now include a ProductTypeId and ProductSubCategoryId
The Product Create view should display two dropdown menus: one for product category and one for product subcategory
Both dropdown menus should contain an initial value prompting the user to select a product category and product subcategory


To Test

git fetch
git checkout es-dropdowns
delete your migrations folder
-delete your database
-reset the path for your database by entering export Bangazon_Db_Path=`"path/to/database.bangazon.db" (on a mac; for windows, use set instead of export)`
enter dotnet ef migrations add WhateverYouWantToCallYourMigration in your terminal
enter dotnet ef database update in your terminal
-reopen your database and double-check that customers, products, product types, and product type subcategories are seeded. Check that products contain a new property (producttypesubcategoryid)
-run through the app to ensure all features still work and above features have been added
